hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -732,12 +735,14 @@ public OpenKeySession openKey(OmKeyArgs args) throws IOException {
 
   private OMResponse handleError(OMResponse resp) throws OMException {
     if (resp.getStatus() != OK) {
+      LOG.error("OMException occurred: {}", resp.getMessage());
```

Comment:
I would suggest remove this log, for every failure should be specifically handled and logged by where it's called in client.